develop/core/tests/test_payment.py

```diff
@@ -2,7 +2,7 @@
 from django.test import TestCase, Client
 from django.urls import reverse
 
-from vendor.models import Invoice, Receipt, Payment
+from vendor.models import Invoice, Receipt, Payment, receipt
```

Comment:
why is lowercase receipt being imported?

Reply:
that is my linter trying to import things manually. I need to turn that off